### PR TITLE
MC-1677 SQL Browser 5.2 docs update

### PR DESCRIPTION
**Pull request by aberatcan**
- Added create mapping section to SQL Browser page.

### CODE REVIEW

File: docs/modules/tools/pages/sql-browser.adoc [hunk 1]
```diff
@@ -20,17 +20,40 @@ Before you can query an existing map in a cluster, you must create a mapping to
...
-To generate the `CREATE MAPPING` statement automatically, click the *Connect Maps* button
-and select the name of an existing map from the dropdown:
+You can generate `CREATE MAPPING` statement automatically using Connector Wizard. You can open it by clicking the *Connector Wizard* button
+and then select the type of the mapping that you want to create from the dropdown:
 
-image:ROOT:SQLBrowserConnectMaps.png[SQL Browser Connect Maps]
+image:ROOT:SQLBrowserConnectWizardMapType.png[Connecter Wizard Connector Type]
+
+=== Create Mapping for Hazelcast Map
+To create mapping for *Hazelcast Map*, after selecting the connector type as `Create Mapping for Hazelcast Map`, select the name of an existing map from the dropdown:
```
Comment (promanenko) [inline docs/modules/tools/pages/sql-browser.adoc:29]: only maps that don't have any mappings are shown in the dropdown

File: docs/modules/tools/pages/sql-browser.adoc [hunk 2]
```diff
@@ -20,17 +20,40 @@ Before you can query an existing map in a cluster, you must create a mapping to
 This statement maps the data structure to a table that can be queried with Hazelcast SQL.
 You can type this statement into the editor, or you can generate it automatically.
 
-To generate the `CREATE MAPPING` statement automatically, click the *Connect Maps* button
-and select the name of an existing map from the dropdown:
+You can generate `CREATE MAPPING` statement automatically using Connector Wizard. You can open it by clicking the *Connector Wizard* button
+and then select the type of the mapping that you want to create from the dropdown:
```
Comment (Serdaro) [inline docs/modules/tools/pages/sql-browser.adoc:24]: You can generate `CREATE MAPPING` statements automatically using Management Center's connector wizard. Using this wizard, you can create mappings with the connectors for Hazelcast Platform maps, Kafka, and other file systems. Launch the wizard by clicking the **Connector Wizard** button, and see the following sections for each connector type.
and then select the type of the mapping that you want to create from the dropdown:

File: docs/modules/tools/pages/sql-browser.adoc [hunk 3]
```diff
@@ -20,17 +20,40 @@ Before you can query an existing map in a cluster, you must create a mapping to
...
 
-image:ROOT:SQLBrowserConnectMaps.png[SQL Browser Connect Maps]
+image:ROOT:SQLBrowserConnectWizardMapType.png[Connecter Wizard Connector Type]
+
+=== Create Mapping for Hazelcast Map
+To create mapping for *Hazelcast Map*, after selecting the connector type as `Create Mapping for Hazelcast Map`, select the name of an existing map from the dropdown. Only maps that don't have any mappings are shown in the dropdown:
```
Comment (Serdaro) [inline docs/modules/tools/pages/sql-browser.adoc:29]: (Let's put a new line between the heading and its body content, and make the body content as a bulleted list)

=== Create a Mapping for Hazelcast Maps

. Select the connector type as **Create Mapping for Hazelcast Map**.
. Select the name of an existing map from the dropdown; only maps that don't have any mappings are shown.

File: docs/modules/tools/pages/sql-browser.adoc [hunk 4]
```diff
@@ -20,17 +20,40 @@ Before you can query an existing map in a cluster, you must create a mapping to
...
+image:ROOT:SQLBrowserConnectorWizardMaps.png[SQL Browser Connect Maps]
 
 If you need to change the generated statement, you can use the editor.
 
 NOTE: You cannot generate a `CREATE MAPPING` statement for empty maps or maps that store entries in the `NATIVE` format without indexes.
 
 Click *Confirm & Run*.
 
+=== Create Mapping for Kafka
+To create mapping for *Kafka*, after selecting the connector type as `Create Mapping for Kafka`, fill out the newly opened form. While you are filling this form, editor would be updated appropriately.
```
Comment (Serdaro) [inline docs/modules/tools/pages/sql-browser.adoc:40]: (Let's put a new line between the heading and its body content, and make the body content as a bulleted list)

=== Create a Mapping for Kafka

. Select the connector type as **Create Mapping for Kafka**.
. Fill out the newly opened form. While you are filling this form, the editor should be updated appropriately.

File: docs/modules/tools/pages/sql-browser.adoc [hunk 5]
```diff
@@ -20,17 +20,40 @@ Before you can query an existing map in a cluster, you must create a mapping to
...
+
+image:ROOT:SQLBrowserConnectorWizardKafka.png[SQL Browser Connector Wizard Kafka]
+
+If you need to change the generated statement, you can use the editor.
+
+Click *Confirm & Run*.
+
+=== Create Mapping for a File System
+To create mapping for a *File System*, after selecting the connector type as `Create Mapping for a File System`, fill out the newly opened form. While you are filling this form, editor would be updated appropriately.
```
Comment (Serdaro) [inline docs/modules/tools/pages/sql-browser.adoc:49]: (Let's put a new line between the heading and its body content, and make the body content as a bulleted list)

=== Create a Mapping for a File System

. Select the connector type as **Create Mapping for a File System**.
. Fill out the newly opened form. While you are filling this form, the editor should be updated appropriately.

**Review by Serdaro — CHANGES_REQUESTED** [review]
Minor editing suggestions.

**Review by Serdaro — APPROVED** [review]

**Pull request merged**